recipes/yaclib/all/patches/0002-Add-install-commands.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index d58e394..b2df0a3 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -32,6 +32,11 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+ list(APPEND CMAKE_MODULE_PATH "${YACLIB_SOURCE_DIR}/cmake")
+ include(yaclib_flags)
+ 
```

Comment:
Can you make PRs for both patches to yaclib?

For install only needs to make it under options
YACLIB_INSTALL which false by default

Reply:
With pleasure!